### PR TITLE
fix(ui): script v2 feedback

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Cosmetic fixes

- Fix color of Patch tag
- Fix sorting of endpoints groups
- Fix size of upgrade bloc

### CODE REVIEW

**Review by TBonnin — APPROVED** [review]

File: packages/webapp/tailwind.config.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ module.exports = {
 
                 'orange-light': '#FFD19A',
                 'orange-base': '#EC9455',
-                'orange-base-35': '#F5AE73',
+                'orange-base-35': '#f5ae7359',
```
Comment (khaliqgant) [inline packages/webapp/tailwind.config.js]: Is this one too many chars? And should be uppercase

Comment (bodinsamuel) [reply packages/webapp/tailwind.config.js]: it's hex with transparency

Comment (khaliqgant) [reply packages/webapp/tailwind.config.js]: ah, alpha, thanks

**Review by khaliqgant — APPROVED** [review]

**Comment by bodinsamuel** [comment]
@khaliqgant added back v1 flow as quick and dirty fix 
Screenshot 2024-09-11 at 10 38 15
Screenshot 2024-09-11 at 10 38 20

**Pull request merged**